src/CoreBundle/Resources/config/services.xml

```diff
@@ -133,16 +133,16 @@
             <argument type="service" id="chameleon_system_core.util.input_filter" />
         </service>
 
-        <service id="chameleon_system_core.ChameleonSessionManager" alias="chameleon_system_core.session.chameleon_session_manager">
+        <service id="chameleon_system_core.ChameleonSessionManager" alias="chameleon_system_core.session.chameleon_session_manager" public="true">
             <deprecated>The "%service_id%" service is deprecated since Chameleon 6.2.0 - use chameleon_system_core.session.chameleon_session_manager instead.</deprecated>
         </service>
 
-        <service id="session" synthetic="true" />
+        <service id="session" synthetic="true" public="true"/>
 
-        <service id="chameleon_system_core.chameleon_controller" synthetic="true" />
+        <service id="chameleon_system_core.chameleon_controller" synthetic="true" public="true"/>
 
-        <service id="chameleon_system_core.module_resolver" class="ChameleonSystem\CoreBundle\ModuleService\ModuleResolver">
-            <argument type="service" id="service_container" />
+        <service id="chameleon_system_core.module_resolver" class="ChameleonSystem\CoreBundle\ModuleService\ModuleResolver" public="true">
+            <argument /> <!-- Service locator will be set by ChameleonModulePass -->
```

Comment:
The Service locator?

Reply:
Not the Chameleon ServiceLocator but a Symfony service locator which is a subset of the full service container (see https://symfony.com/doc/current/service_container/service_subscribers_locators.html).

Reply:
Then maybe call it "Symfony service locator"?

Reply:
I think it's self-explanatory as soon as we're a bit more familiar with Symfony 3, as it makes neither sense nor is it possible to inject the Chameleon ServiceLocator (there is no service definition for it).